observer.go

```diff
@@ -57,10 +58,56 @@ func initTLS() *tls.Config {
 
 	var clientPool []tls.Certificate
 	if len(config.Aerospike.CertFile) > 0 || len(config.Aerospike.KeyFile) > 0 {
-		// client Cert
-		cert, err := tls.LoadX509KeyPair(config.Aerospike.CertFile, config.Aerospike.KeyFile)
+
```

Comment:
This function has become too long. I think it's not a bad idea to move the reading of the file out to a separate function.

Reply:
Done. moved all certificate file reads into a function `readCertFile()`